.github/workflows/ci-daily-local.yml

```diff
@@ -0,0 +1,44 @@
+name: CI - FAST Local Testing
+
+jobs:
+  build_local_linux:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Lerna
+      run: yarn global add lerna
+    
+    - name: Upgrade TypeScript

```

Comment:
This was necessary to upgrade the docker container when running locally.  This will never run except locally.